src/Request.php

```diff
@@ -47,8 +32,10 @@ protected function validator(): IlluminateValidator
      */
     protected function getInput(string $key, $default = null)
     {
-        if (array_key_exists($key, $this->input)) {
-            return $this->input[$key];
+        $input = $this->json()->all();
```

Comment:
`$this->input()` seems to work now that you registered the resolving callback. I think it would be nice to leverage this method since it would replace the need for this `getInput` method. This method kind of reimplements the already existing one.